src/reporter/index.ts

```diff
@@ -267,7 +267,10 @@ export default class Reporter {
                 id:   testRun.test.fixture.id
             },
             command: formatCommand(command, result),
-            browser: testRun.controller.browser,
+            // TestController doesn't have the browser property in its definition.
+            // It is dynamically added later.
+            // @ts-ignore
```

Comment:
IMHO, this property should be declared in the TestController interface, even if it is not assigned in the constructor, but some time later in the external code.